app/views/metrics/_tile_with_chart.html.erb

```diff
@@ -1,4 +1,7 @@
-<%= polaris_card(title: t("metrics.tiles.#{tile.handle}", default: tile.handle.to_s.humanize)) do |card| %>
+<%= polaris_card(title: t("metrics.tiles.#{tile.handle}", default: tile.handle.to_s.humanize), classes: 'relative') do |card| %>
+  <div class="loading-overlay hidden Polaris-LegacyCard">
+    <%= polaris_spinner(accessibility_label: "Loading chart data...") %>
```

Comment:
Great to see accessibility labels used. Can you please add `i18n` for the string? Most of the app uses i18n for user facing strings, so would be great to have it here too.